### ISSUE TITLE
Method to add distance to the trajectory DataFrame

### ISSUE THREAD

**Issue by bkowshik** [anitagraser/movingpandas#125]
Similar to the `add_direction` and `add_speed` methods, it will be useful to add `add_distance` as a method that computes distance between successive points of the trajectory. A distribution of `distance`, looking at large and small distance segments could be useful to get an understanding of the datasets.

Ref: https://movingpandas.readthedocs.io/en/master/trajectory.html

**Comment by anitagraser** [comment]
That's correct. Distance calculation is already implemented. This feature should therefore be straightforward to implement. Please provide unit tests with the PR.
